oam/install_scripts/syslogSetup.sh

```diff
@@ -212,20 +223,23 @@ if [ ! -z "$syslog_conf" ] ; then
 			if [ $? -eq 0 ]; then
 				if [ -f ${syslog_conf}.columnstoreSave ] ; then
 					#uninstall the syslog for ColumnStore logging
-					$SUDO v -f ${syslog_conf} ${syslog_conf}.ColumnStoreBackup
-					$SUDO mv -f ${syslog_conf}.columnstoreSave ${syslog_conf} >/dev/null 2>&1
+					 v -f ${syslog_conf} ${syslog_conf}.ColumnStoreBackup
```

Comment:
I assume this is supposed to be 'mv' not 'v', and that this has been broken for a while.